### DISCUSSION TITLE
使用thinkjs-cli创建的typescrpit项目运行npm run compile会导致项目报错

### DISCUSSION THREAD

**Discussion by SuperLeven** [thinkjs/thinkjs#1717] [Q&A]
如题，有大神知道怎么解决吗？
![Uploading 截屏2023-05-06 17.49.00.png…]()

**Comment by SuperLeven** [comment]
![截屏2023-05-06 17 49 48](https://user-images.githubusercontent.com/75765099/236616998-8c7b71b4-ac6f-4402-84a1-6e00c9a0c67b.png)

**Reply by zhangbowy** [reply]
先 npm i 一下
